datacite-transform/build.gradle

```diff
@@ -1,41 +1,80 @@
-import java.nio.file.Files
-import java.nio.file.Paths
-
 plugins {
     id 'java'
-    id "org.unbroken-dome.xjc" version "2.0.0"
+    id("io.github.alexpercont.xjc-plugin") version "1.1"
```

Comment:
This is not necessarily the best choice, but it is the most recent and supports jakarta.

The download plugin is a common choice.

Reply:
I think @ketilaa have a point regarding the xsd and if we also don't need to rely on a plugin with short track record and only one contributor, that would be nice. If download is not a requirement, would it be easier to find an alternative?

Reply:
We changed the plugin to another one-person (with two personalities) project, but with *more users*